### ISSUE TITLE
Weird behaviour with export

### ISSUE THREAD

**Issue by rafaeleyng** [javascript-obfuscator/javascript-obfuscator#791]
Hello

I was having trouble using this (with the Webpack loader) in my project. I ended up finding that the problem was in some files where I had code like this:

```js
import funcA from './funcA'
import funcB from './funcB'

export {
  funcA,
  funcB,
}
```

With that, some other module that would try to import and use `funcA` would not work (would get `undefined` or something like it). It seems that in one point the identifiers were changed but not in the other point.

When I changed the code to this, it worked:
```js
export { default as funcA } from './funcA'
export { default as funcB } from './funcB'
```

I'm ok with doing this change, but I would like to better understand whether I'm doing something wrong, and why making this change solved the problem.

Thanks!

**Comment by sanex3339** [comment]
Hmm. Obfuscated code looks like that:
```
import _0x99b4d7 from './funcA';
import _0x3e429c from './funcB';
export {
    _0x99b4d7,
    _0x3e429c
};
```

So it renames the imported module identifiers.
Have to think about how to avoid this.

**Comment by sanex3339** [comment]
So, the idea is to convert all exports
```
export {foo};
```

to the
```
export {foo as foo};
```

view.

After identifiers rename this construction becomes
```
const _0x485b3c = 1;
export {_0x485b3c as foo};
```

**Comment by sanex3339** [comment]
PR: https://github.com/javascript-obfuscator/javascript-obfuscator/pull/792

Can you try it?

**Comment by sanex3339** [comment]
Merged into `master`. Please try it from `master` branch.
I'll release this fix later today.

**Comment by sanex3339** [comment]
Released as `2.6.3`. Please try it.

**Comment by rafaeleyng** [comment]
Sorry for the delay, somehow I messed up my notification settings and didn't see this earlier.

I've tested the latest `2.8.0` and it works perfectly. Thanks for the quick response!
